README.md

```diff
@@ -6,56 +6,27 @@ We use a template repository approach for maintainability.
 
 ## Creating a repository for an individual training workshop
 
-First, you will need to create a new repository.
+First, you will need to create a new repository using the [`Use the template` button](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template).
+This will create a new repository with the same layout as this template repository under the given organization or user you choose to administer the repository; if you are a Data Lab member running an internal training workshop, the `AlexsLemonade` organization should administer the new repository.
```

Comment:
You will be set as an administrator (and can add others), but the org is the Owner, which is what the menu you select from shows.
```suggestion
This will create a new repository with the same layout as this template repository under the user or organization you choose as the owner of the repository; if you are a Data Lab member running an internal training workshop, the `AlexsLemonade` organization should be the owner.
```